src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -43,6 +43,10 @@
     <TSqlScript Include="Features\Schema\Sql\Tables\*.sql" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This gets automatically added with every version increase, but I don't think it is needed.